.expeditor/generate_verify_pipeline.sh

```diff
@@ -14,7 +14,7 @@ set -euo pipefail
 # Group them by plan and use that as the unit of work in
 # downstream steps.
 plans_changed() {
-  git diff --name-only "${BUILDKITE_BRANCH}" "$(git merge-base "${BUILDKITE_BRANCH}" master)" \
+  git diff --name-only ${TEST_BRANCH:-HEAD} "$(git merge-base ${TEST_BRANCH:-HEAD} origin/master)" \
```

Comment:
I've seen some unexpected behaviour when relying on `git diff` relative to origin master - I think the buildkite agents may have a old revision of the repo already on the disk with HEAD pointing to an older commit than you'd expect.
I think I worked around it by doing a `git fetch origin master` to ensure the master branch on disk is the current HEAD ref for the repo.
cc @tduffield